API-reference.md

```diff
@@ -196,6 +196,22 @@ Adds a telemetry initializer to the collection. Telemetry initializers will be c
 before the telemetry item is pushed for sending.
 If one of the telemetry initializers returns false or throws an error, then the telemetry item will not be sent.
 
+### getSender
```

Comment:
@MSNev do you think this is the proper place to add the new API instruction? Thanks.

Reply:
Yep, when we check this in lets also update the Issue with a link to this file so it can be more easily found.